package.json

```diff
@@ -38,7 +38,7 @@
     "url": "https://github.com/calibreapp/react-live-chat-loader.git"
   },
   "peerDependencies": {
-    "react": "^16.14.0 || ^17.0.0"
+    "react": "^16.14.0 || ^17.0.0 || ^18.0.0",
```

Comment:
According to Vercel's build, this is invalid 

<img width="1226" alt="image" src="https://user-images.githubusercontent.com/924/180682030-aa76c6c0-ab30-47cc-85fc-adef4ebd4a66.png">